src/components/defaultLayout/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Account from '../account';
+import PrivateRoute from '../privateRoute';
+
+const DefaultLayout = ({ component: Component, ...rest }) => (
+  <PrivateRoute {...rest} component={ matchProps => (
+    <main>
+      <Account />
```

Comment:
The Links component was moved on @slaweet request. I think that the performance change here is negligible especially if you will move the logic from account component and keep it dumb and allow only render the passed props.  I do not know how the app will be looked after rebranding, and I think that the layout can help combine some components for differents views.
If you think that is a bad solution than I can return the nested route.